l10n_fi_reconcile/models/account_bank_statement.py

```diff
@@ -109,7 +109,21 @@ def _get_auto_reconcile_new_aml_dicts(self, match_recs):
         auto reconciling complex situations, e.g. cash discounts.
         This stub method may be extended to handle those situations.
         """
-        return None
+        return []
```

Comment:
Might this change be part of a different change? It doesn't seem to have anything to do with the rest of this change.

Reply:
Can we split this diff line to a different commit?

Reply:
Split done. It is sort of related to rest of the changes, as it improves extensibility and what it returns is passed to `is_valid_finnish_match` for validation.